docker/update-table-reference/pyproject.toml

```
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "updatereference"
```

@TruongQuangSB We should stick with a common 

```suggestion
name = "update-table-reference"
```